app/models/book.rb

```diff
@@ -0,0 +1,2 @@
+class Book < ActiveRecord::Base
```

Comment:
Yes, since we don't have plans to have any kind of administrative interface it's okay for me to keep it clear of validations for awhile.